libs/shared/ui/src/lib/components/badge/badge.spec.tsx

```diff
@@ -0,0 +1,9 @@
+import { renderWithProviders, screen } from '@qovery/shared/util-tests'
+import { Badge } from './badge'
+
+describe('Badge', () => {
```

Comment:
Can we have a snapshot for all variants or you'll implement it after?

Reply:
I can do it but it's a combinatorial issue, because we have `size` `color` and `variant` params to test.
I didn't see the point of testing all those visual variants so I've only tested the functional behavior.